storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/BlindedBlockMigration.java

```diff
@@ -97,7 +107,7 @@ private void moveHotBlocksToBlindedStorage() {
             blindedUpdater.addBlindedBlock(block, block.getRoot(), spec);
             unblindedUpdater.deleteUnblindedHotBlockOnly(block.getRoot());
             counter++;
-            if (counter % 128 == 0 || counter == countBlocks) {
+            if (counter % 32 == 0 || counter == countBlocks) {
```

Comment:
Do we need to make it log progress this often? Not against it but it seems surprisingly frequent.

Reply:
just for hot blocks, can drop it back but at 128 almost never will see... dont have a strong opinion... was helpful debugging